application/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" monitorInterval="30">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <RollingFile name="File" fileName="logs/app.log" filePattern="logs/app-%d{dd-MM-yyyy}.log.gz"
```

Comment:
the test should probably not log into the same file than the real code. double check the paths/file names and adjust if necessary